src/main.rs

```diff
@@ -869,4 +914,213 @@ mod tests {
             assert_eq!(response.status(), StatusCode::BAD_REQUEST);
         }
     }
+
+    mod config {
+        use crate::ConfigurationFile;
+        use cryptography::x509::attr::Attribute;
+        use cryptography::x509::request::{CertReq, ExtensionReq};
+        use cryptography::x509::Certificate;
+        use der::Decode;
+        use sgx_validation::quote::traits::ParseBytes;
+        use sgx_validation::quote::Quote;
+        use snp_validation::Report;
+        use snp_validation::{Evidence, Snp};
+
+        const STEWARD_CONFIG: &str = include_str!("../testdata/steward.toml");
+        const ICELAKE_CSR: &[u8] =
+            include_bytes!("../crates/sgx_validation/src/icelake.signed.csr");
+        const MILAN_CSR: &[u8] = include_bytes!("../crates/snp_validation/src/milan.signed.csr");
```

Comment:
No, this does not belong in production binaries

Reply:
These are in the unit tests.

Reply:
May be cleaned up when https://github.com/profianinc/steward/issues/134 is addressed.